tpp-sample/src/sample/CancelTokenSample.cs

```diff
@@ -20,7 +20,21 @@ public static TokenOperationResult CancelAccessToken(TppMember grantee, string t
             Token accessToken = grantee.GetTokenBlocking(tokenId);
             // Cancel access token.
             return grantee.CancelTokenBlocking(accessToken);
+        }
+
+        /// <summary>
+        /// Cancels the transfer token.
+        /// </summary>
+        /// <param name="payee">payee Token member</param>
+        /// <param name="tokenId">token ID to cancel</param>
+        /// <returns>operation result</returns>
+        public static TokenOperationResult CancelTransferToken(TppMember payee, string tokenId)
```

Comment:
> it is better to have only one cancel token sample since they are exactly the same.

It is there is Java SDK as well, so we added here the same way.